src/components/DiffView/index.tsx

```diff
@@ -265,6 +284,42 @@ export class DiffViewBase extends React.Component<Props> {
     return <div className={styles.extraMessages}>{messages}</div>;
   };
 
+  renderGutter = ({
+    change,
+    renderDefault,
+    side,
+    wrapInAnchor,
+  }: RenderGutterParams) => {
+    const { enableCommenting, version } = this.props;
+
+    const { isDelete, lineNumber } = change;
```

Comment:
This [part of the docs](https://github.com/otakustay/react-diff-view/#key-of-change) suggests maybe you should be using `oldLineNumber` for normal changes. Could you try to comment on some normal diffs and compare it in browse mode to make sure they appear on the right lines? It might be fine; I have a vague recollection that `lineNumber` and `oldLineNumber` are identical....maybe.

Reply:
I checked, and a comment added in Compare mode on a normal line does also appear in Browse mode for the same file and version, so I guess the current code is okay.